### PR TITLE
Citation and referencing guidelines

### PR DESCRIPTION
**Pull request by gwbres**

### CODE REVIEW

**Comment by gwbres** [comment]
@lnicola do we have something similar either in the portal or other main crates of the ecosystem ?

**Review by lnicola — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -79,10 +79,17 @@ from raw uBlox GNSS receiver frames. This application is work in progress at the
...
+* [Hifitime](https://github.com/nyx-space/hifitime): Precise Timing, Timescales, ...
+* [CGGTTS](https://github.com/gwbres/cggtts): Common View Time Transfer
+- [RTK-RS](https://github.com/rtk-rs/gnss-rtk): Precise Positioning
+* [GNSS definitions in Rust](https://github.com/rtk-rs/gnss): GNSS library
+
+## Citation and referencing
+
+If you need to reference this work, please use the following model:
+
+`GeoRust RINEX Team (2023), RINEX: analysis and processing (Apache2/MIT), https://georust.org`
```
Comment (lnicola) [inline README.md:92]:
```suggestion
`GeoRust RINEX Team (2023), RINEX: analysis and processing (Apache-2/MIT), https://georust.org`
```

**Comment by lnicola** [comment]
I remember a discussion about assigning DOIs to GeoRust projects, you can ask on Discord.

**Comment by gwbres** [comment]
@lnicola 

It is not in place at the moment, but solutions exist to generate DOIs. Let's add this entry and I may try to generate one when publishing next release

**Pull request merged**